dist/morphext.js

```diff
@@ -36,22 +38,35 @@
     Plugin.prototype = {
         _init: function () {
             var $that = this;
+            this.position = 0;
+            this.loops = 0;
             this.phrases = [];
 
             this.element.addClass("morphext");
 
             $.each(this.element.text().split(this.settings.separator), function (key, value) {
                 $that.phrases.push($.trim(value));
+                
             });
-
+            this.loops = this.settings.loop * this.phrases.length;   
+                        
             this.index = -1;
             this.animate();
-            this.start();
+            this.start();                  	
+           
         },
         animate: function () {
+            this.position++;
             this.index = ++this.index % this.phrases.length;
             this.element[0].innerHTML = "<span class=\"animated " + this.settings.animation + "\">" + this.phrases[this.index] + "</span>";
+            this.element[0].classList.add(this.settings.activeClass);
```

Comment:
I'm not sure if it's necessary for us to `add` in every iteration. I feel like there should be a more elegant solution (e.g. setting a base class on the parent element, and having its style get overwritten by `morphext` on line 41/45).